src/core/server/saved_objects/import/import_saved_objects.ts

```diff
@@ -80,12 +81,20 @@ export async function importSavedObjectsFromStream({
   if (createNewCopies) {
     importIdMap = regenerateIds(collectSavedObjectsResult.collectedObjects);
   } else {
+    importIdMap = await regenerateIdsWithReference({
```

Comment:
Since `importSavedObjectsFromStream ` is a function already used in OSD main, will this logic have a new influence if feature flag is turned off?